mungegithub/features/repo-updates.go

```diff
@@ -167,7 +168,7 @@ func (o *RepoInfo) EachLoop() error {
 
 // AddFlags will add any request flags to the cobra `cmd`
 func (o *RepoInfo) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVar(&o.kubernetesDir, "kubernetes-dir", "./gitrepos/kubernetes", "Path to git checkout of kubernetes tree")
+	cmd.Flags().StringVar(&o.kubernetesDir, "kubernetes-dir", "", "Path to git checkout of kubernetes tree")
```

Comment:
Yes, that PR is in flight as well. :) https://github.com/kubernetes/contrib/pull/1394/